native/game_backend/src/map.rs

```diff
@@ -22,20 +22,20 @@ pub fn hit_boxes_collide(center1: &Position, center2: &Position, size1: u64, siz
 pub fn in_cone_angle_range(
     center_player: &Player,
     target_player: &Player,
-    max_distance: u64,
+    max_distance: f32,
     cone_angle: f32,
 ) -> bool {
     // TODO: Take into consideration `size` attribute of Player
-    let squared_x = (center_player.position.x - target_player.position.x).pow(2) as f64;
-    let squared_y = (center_player.position.y - target_player.position.y).pow(2) as f64;
+    let squared_x = ((center_player.position.x - target_player.position.x) as f64).powi(2);
+    let squared_y = ((center_player.position.y - target_player.position.y) as f64).powi(2);
     let target_distance = (squared_x + squared_y).sqrt();
```

Comment:
Do we still need this `as f64`?

Reply:
Yes, since squaring the number might not Fit in an f32